### PR TITLE
Change table style - Closes #535

### LINKED ISSUE #535: Change table representation to be able to handle a lot of data
### Description

Currently, if we have a lot of keys, it will widely spread the header, and small window size will not be able to handle it.

**Expected behavior:** [What you expect to happen]
It should be able to show all the information on the smaller window.

### Versions

1.0.0

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?
When the header gets too big, on table format it cannot be displayed well.

### How did I fix it?
Make the table vertical.

### How to test it?
```
lisky set json false
lisky list delegates genesis_5, genesis_6
```

### Review checklist

* The PR solves #535
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/specs/utils/tablify.js [hunk 1]
```diff
@@ -33,40 +33,32 @@ describe('tablify util', () => {
...
+				then.theReturnedTableShouldHaveARowWithTheObjectKeyAndStringifiedNestedValues,
 			);
 		});
 	});
 	Given('a deeply nested object', given.aDeeplyNestedObject, () => {
 		When('the object is tablified', when.theObjectIsTablified, () => {
 			Then(
-				'it should throw print error "Output cannot be displayed in table format: Maximum object depth of 3 was exceeded. Consider using JSON output format."',
-				then.itShouldThrowPrintError,
+				'the returned table should have a row with the object key and stringified nested values',
```
Comment (willclarktech) [inline test/specs/utils/tablify.js]: What about arrays e.g. of votes? This one only has 4 and is already off the screen.
```
lisky> get transaction 3511360482635196713
╔════════════════════╤══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ id                 │ 3511360482635196713                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ height             │ 340572                                                                                                                                                                                                                                                                                   ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ blockId            │ 5452350734325765704                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ type               │ 3                                                                                                                                                                                                                                                                                        ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ timestamp          │ 63398593                                                                                                                                                                                                                                                                                 ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ senderPublicKey    │ 34682d411cdccb22f7e5f1158279cec04a43caa1ed41c2a511f2369709fcd6be                                                                                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ senderId           │ 5057926541223936699L                                                                                                                                                                                                                                                                     ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ recipientId        │ 5057926541223936699L                                                                                                                                                                                                                                                                     ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ recipientPublicKey │ 34682d411cdccb22f7e5f1158279cec04a43caa1ed41c2a511f2369709fcd6be                                                                                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ amount             │ 0                                                                                                                                                                                                                                                                                        ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ fee                │ 100000000                                                                                                                                                                                                                                                                                ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ signature          │ 31890206cc2166e2d2a8c8ba56b4c4f383f8e6adac555ee2d7b5a92075349c8c71aeef683d30c46197410dc8ac6e2ec5a1748c16cdae6dc3783426d859193501                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ signatures         │                                                                                                                                                                                                                                                                                          ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ confirmations      │ 789                                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ asset              │ votes: ["+ef63d95c454a52d91d05a9bc072a7350e55157974afe5ae66ad3cbc27fa25360","+2a0183a7343a7bc29c84bc37a19ed149c478b82189225b47795b873a9716c569","-374ce08c19f30551c1485e1213b10658646a7ed69779c446ffba23c367b404e5","-31cc9655335ba9cf01b8393311bb12bff8ea2a2c219e10d819734ce95e5c15f1"] ║
╚════════════════════╧══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
```

File: test/steps/printing/3_then.js [hunk 2]
```diff
@@ -134,90 +134,68 @@ export function theReturnedTableShouldHaveNoRows() {
...
-	return expect(returnValue.options)
-		.to.have.property('head')
-		.eql(testArrayKeysResult);
+	return Object.entries(testObject).forEach(([key, value], arrayKey) => {
+		const strValue =
+			typeof value === 'object'
+				? Object.entries(value)
+						.map(([vKey, vValue]) => `${vKey}: ${JSON.stringify(vValue)}`)
+						.join('\n')
+				: value;
```
Comment (willclarktech) [inline test/steps/printing/3_then.js]: Is this always a string?

**Comment by willclarktech** [comment]
I think this would have to be semver-major just in case people are parsing table output for some reason.

File: test/specs/utils/tablify.js (hunk 1 above)
Comment (shuse2) [reply test/specs/utils/tablify.js]: votes...!!!

File: test/steps/printing/3_then.js (hunk 2 above)
Comment (shuse2) [reply test/steps/printing/3_then.js]: At this point, it's either string or number, which are both acceptable.

**Pull request merged**